build_runner/lib/src/entrypoint/options.dart

```diff
@@ -131,12 +132,14 @@ class SharedOptions {
 /// Options specific to the `serve` command.
 class ServeOptions extends SharedOptions {
   final String hostName;
+  final bool liveReload;
   final bool hotReload;
   final bool logRequests;
   final List<ServeTarget> serveTargets;
 
   ServeOptions._({
```

Comment:
We should add some validation here that both `hotReload` and `liveReload` are not true.